reinforcement_learning/examples/rl_sim_cpp/rl_sim.cc

```diff
@@ -21,37 +22,35 @@ int rl_sim::loop() {
     const auto context_features = p.get_features();
     const auto action_features = get_action_features();
     const auto context_json = create_context_json(context_features,action_features);
-    const auto req_id = create_uuid();  
+    const auto req_id = create_event_id();  
     r::api_status status;
 
     // Choose an action
     if ( _rl->choose_rank(req_id.c_str(), context_json.c_str(), response, &status) != err::success ) {
       std::cout << status.get_error_msg() << std::endl;
-      return -1;
+      continue;
```

Comment:
I am not sure, @ataymano would need to answer this

Reply:
It may be because we saw occasional network failures and exiting the entire run didn't make sense but to just skip that attempt.

Reply:
Since rl_sim is a sample app, probably good to exit in case of error here (or at least handle it differently) to promote proper consumption.  In test_cpp  its probably appropriate to continue.